core/src/main/scala/scheduler/DeploymentSummary.scala

```diff
@@ -17,10 +17,28 @@
 package nelson
 package scheduler
 
+import scalaz.Monoid
+import scalaz.Scalaz._
```

Comment:
Can we use the a la carte imports within the scope of a module? I dont like importing these globally in a file